daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/NamedEntity.scala

```diff
@@ -18,12 +17,11 @@ object NamedEntity {
 
   final case class NModDef(
       name: ModuleName,
-      dfns: List[(DottedName, Ast.Definition)],
   ) extends NamedEntity {
 
     def modName: ModuleName = name
 
-    def fullyResolvedName: DottedName = name.toUpperCase
+    val fullyResolvedName: DottedName = name.toUpperCase
```

Comment:
This value gets reused a lot, so it's better for it to be a `val` instead of a `def` in this case, if I understand correctly.